creativecloud/features/genfill/genfill-interactive.js

```diff
@@ -1,75 +1,82 @@
 import { createEnticement } from '../interactive-elements/interactive-elements.js';
 import defineDeviceByScreenSize from '../../scripts/decorate.js';
 
-function handleTransition(pics, index) {
-  pics[index].style.display = 'none';
-  const nextIndex = (index + 1) % pics.length;
-  pics[nextIndex].style.display = 'block';
+async function addEnticement(container, enticement, mode) {
+  const svgUrl = enticement.querySelector('a').href;
+  const enticementText = enticement.innerText;
+  const entcmtEl = await createEnticement(`${enticementText}|${svgUrl}`, mode);
+  entcmtEl.classList.add('enticement');
+  const viewports = ['tablet', 'desktop'];
+  viewports.forEach((v) => {
+    const mDiv = container.querySelector(`.media.${v}-only`);
+    mDiv.insertBefore(entcmtEl.cloneNode(true), mDiv.firstElementChild);
+  });
+}
+
+function generateDaaLL(hText, alt, v) {
+  const altTxt = alt
+    ? `${alt}|Marquee|${hText}`
+    : `Image-${v}|Marquee|${hText}`;
+  return altTxt;
+}
+
+function handleClick(a, v, deviceConfig, hText) {
+  const img = a.querySelector('img');
+  const currIndex = deviceConfig[v].index;
+  const nextIndex = (currIndex + 1) % deviceConfig[v].srcList.length;
+  img.src = deviceConfig[v].srcList[nextIndex];
+  const alt = deviceConfig[v].altList[nextIndex];
+  a.setAttribute('daa-ll', generateDaaLL(hText, alt, v));
+  deviceConfig[v].index = nextIndex;
   return nextIndex;
 }
 
-function startAutocycle(interval, pics, clickConfig) {
-  if (clickConfig.isImageClicked) return;
-  clickConfig.autocycleInterval = setInterval(() => {
-    clickConfig.autocycleIndex = handleTransition(pics, clickConfig.autocycleIndex);
-    if (clickConfig.autocycleIndex === pics.length - 1) {
-      clearInterval(clickConfig.autocycleInterval);
+function startAutocycle(a, autoCycleConfig, viewport, deviceConfig, interval, hText) {
+  if (autoCycleConfig.isImageClicked) return;
+  autoCycleConfig.autocycleInterval = setInterval(() => {
+    handleClick(a, viewport, deviceConfig, hText);
+    if (autoCycleConfig.isImageClicked
+      || deviceConfig[viewport].index === deviceConfig[viewport].srcList.length - 1) {
+      clearInterval(autoCycleConfig.autocycleInterval);
     }
   }, interval);
 }
 
-function handleClick(aTags, clickConfig) {
-  aTags.forEach((a, i) => {
-    a.querySelector('img').removeAttribute('loading');
-    a.addEventListener('click', () => {
-      clickConfig.isImageClicked = true;
-      if (clickConfig.autocycleInterval) clearInterval(clickConfig.autocycleInterval);
-      handleTransition(aTags, i);
-    });
+function processMedia(ic, miloUtil, autoCycleConfig, deviceConfig, v, hText) {
+  const media = miloUtil.createTag('div', { class: `media ${v}-only` });
+  const a = miloUtil.createTag('a', { class: 'genfill-link' });
+  const img = miloUtil.createTag('img', { class: 'genfill-image' });
+  const alt = [deviceConfig[v].altList];
+  [img.alt] = generateDaaLL(hText, alt, v);
```

Comment:
Alt should not be modified value.. daa-ll should be in a tag only.
if (alt) img.alt = alt